backend/store/principal.go

```diff
@@ -236,6 +236,10 @@ func (*Store) listUserImpl(ctx context.Context, tx *Tx, find *FindUserMessage) (
 			userMessage.Roles = append(userMessage.Roles, api.WorkspaceAdmin)
 		}
 
+		if !containsWorkspaceRole(userMessage.Roles) {
+			userMessage.Roles = append(userMessage.Roles, api.WorkspaceMember)
+		}
+
```

Comment:
What about always appending the `WorkspaceMember` role? What will be the difference?

Reply:
This will change the UX, all users will render the "Workspace Member" in the UI, little weird